.github/workflows/oss-merge-trigger.yml

```diff
@@ -8,7 +8,7 @@ on:
       - closed
     branches:
       - main
-      - 'release/*.*.x'
+      - release/**
```

Comment:
I specifically went with `release/**` for consistency with [our other workflows that trigger on the same set of branches](https://github.com/search?q=repo%3Ahashicorp%2Fconsul+%22release%2F**%22+language%3AYAML&type=code&l=YAML)